apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/CCWProcedure.java

```diff
@@ -1,11 +1,11 @@
-package gov.cms.bfd.server.war.stu3.providers;
+package gov.cms.bfd.server.war.commons;
 
 import java.time.LocalDate;
 import java.util.Objects;
 import java.util.Optional;
 
 /** Models a ccwProcedure code entry in a claim. */
-final class CCWProcedure extends IcdCode {
+public final class CCWProcedure extends IcdCode {
```

Comment:
Were you still wanting to rename this, CCWProcedure?

Reply:
Not in this iteration.